### DISCUSSION TITLE
FastCGI Cache and effects on WP plugins ability to monitor traffic.

### DISCUSSION THREAD

**Discussion by stuffeh** [Pierre-Lannoy/wp-decalog#34] [General]
I have a question about wordpress plugin ability to detect and thus log a server's ability to monitor traffic when page caching is enabled. 

Logically, I think your plugins such as vibes and the console wouldn't be able to see the requests since FastCGI cache would see the page request and serve up cached html version of the page, preventing wp and any plugins on wp from seeing the requests. Thus, the data displayed by those plugins would be incorrect. 

But I'm relatively new to wp and to your plugins, so I'm hoping there's a way around this.

**Comment by Pierre-Lannoy** [comment]
Hello @stuffeh ! ;)
